ouroboros-consensus/src/Ouroboros/Consensus/Ledger/SupportsMempool.hs

```diff
@@ -32,6 +33,19 @@ data family GenTx blk :: Type
 -- error type as when updating it with a block
 type family ApplyTxErr blk :: Type
 
+data WhetherToForgive
+  = DoForgive
```

Comment:
Why not just `Forgive`? My eyes will have better way of differentiating between those two values.

Reply:
I like the strict duality between `Do` and `DoNot`. But I agree with you that it's easier on the eyes, so I'm dropping it 👍